python/paddle/fluid/imperative/checkpoint.py

```diff
@@ -0,0 +1,241 @@
+# Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+from paddle.fluid.wrapped_decorator import signature_safe_contextmanager
+
+import os
+import contextlib
+import paddle.fluid as fluid
+from paddle.fluid import core
+from paddle.fluid.framework import Variable
+from paddle.fluid.executor import Executor
+
```

Comment:
Add ```__all__``` here, what is the interface api exported to user?

Reply:
added